__tests__/__snapshots__/generic.test.js.snap

```diff
@@ -2,6 +2,6 @@
 
 exports[`Generic functionality throws when too few parameters are sent in 1`] = `"First argument must be a function"`;
 
-exports[`Generic functionality throws when too many parameters are sent in 1`] = `"Too many parameters. The function get dll version do not take 2 parameters."`;
+exports[`Generic functionality throws when too many parameters are sent in 1`] = `"Too many parameters. The function get library version do not take 2 parameters."`;
```

Comment:
s/do not/does not/

Not really important since this is a test script, so meh.

Reply:
Not important I guess, but it is the same message you would get if you check the errormessage as well, so will fix the root cause.